kubernetes/pods.json

```diff
@@ -9,9 +9,6 @@
   "requiredProperties": [
     "kubernetes_pod_name"
   ],
-  "filterProperties": [
```

Comment:
Are you sure this should be removed? Note that it's different than `requiredProperties`.